internal/cmd/api-key/command_delete.go

```diff
@@ -56,11 +52,19 @@ func (c *command) delete(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func (c *command) validateArgs(cmd *cobra.Command, args []string) error {
+func (c *command) confirmDeletion(cmd *cobra.Command, args []string) error {
 	describeFunc := func(id string) error {
 		_, _, err := c.V2Client.GetApiKey(id)
```

Comment:
Why are we ignoring the `detail` field of the HTTP response (second argument) here again?

Reply:
I'm only looking to see whether there was an error or not. All id's that return an error here are reported together in a different error message along with a suggestion to use the list command to view valid ids.

Reply:
Yeah I'm not using the errors from the describe calls; I'm only looking at whether or not an error happened. What I meant about combined error message was this:
```
Error: connectors "lcc-456", "lcc-789", and "lcc-101112" not found

Suggestions:
    List available connectors with `confluent connect cluster list`.
```